src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -600,6 +607,41 @@ def subvolume_exists(self, **kwargs):
                 ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def set_earmark(self, **kwargs):
+        ret       = 0, "", ""
+        volname   = kwargs['vol_name']
+        subvolname = kwargs['sub_name']
+        groupname = kwargs['group_name']
+        force     = kwargs['force']
+        earmark   = kwargs['earmark']
```

Comment:
Not pep8 style?

Reply:
yeah that's something I don't think is followed for this file & so I've avoided many pep8 style/annotations

Reply:
OK.